utils/configen/mkconfig.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+
+	"github.com/BurntSushi/toml"
+	"github.com/coniks-sys/coniks-go/keyserver"
```

Comment:
This util belongs in `keyserver/utils/`.  Generally, utils probably doesn't want to depend on other subpackages.